### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by dawiddylowicz**
We are bumping version to 1.0.0 because:
* SDK has already been implemented in production integrations
* changes introduced in this version are not backwards-compatible
* we are following Semantic Versioning for consistency

### Changed
- Public: Support uploading captures to the Onfido API through use of new SDK tokens (JWT v2)

### Removed
- Public: Face no longer supports PDF upload in order to align with the Onfido API.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://183-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -1,21 +1,18 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "0.15.1",
+  "version": "1.0.0-rc.1",
```
Comment (dawiddylowicz) [inline package.json]: I'll update this to 1.0.0 before releasing and merging, once PR reviewed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 
 ## [next-version]
 
+## [1.0.0]
+
+We are bumping version to 1.0.0 because:
+* SDK has already been implemented in production integrations
+* changes introduced in this version are not backwards-compatible
+* we are following Semantic Versioning for consistency
```
Comment (hjheath) [inline CHANGELOG.md]: Point 1 is sensible. Point 2 & 3 are implicit - they are not the reason we are bumping to 1.0.0. I would remove them then add the main reason we are bumping to 1.0.0 which is that we now integrate with the Onfido API.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,20 +1,36 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-This project adheres to the Node default version scheme, meaning It's safe to use with the caret ^ dependency definition.
-The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).
-Breaking changes result in a different major. UI changes that might break customizations on top of the sdk, will be treated as breaking changes too.
+
+This change log file is based on best practices from [Keep a CHANGELOG](http://keepachangelog.com/).  
+This project adheres to the Node default version scheme, meaning it is safe to use with the caret ^ dependency definition. The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).
```
Comment (hjheath) [inline CHANGELOG.md]: I think the line on Node version scheme has too much detail. I would just say `This project adheres to the Node default version scheme.` and link to it. I would also put it below Semver

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,20 +1,36 @@
...
+We are bumping version to 1.0.0 because:
+* SDK has already been implemented in production integrations
+* changes introduced in this version are not backwards-compatible
+* we are following Semantic Versioning for consistency
+
+### Changed
+- Public: Support uploading captures to the Onfido API through use of new SDK tokens (JWT v2)
+
+### Removed
+- Public: Face no longer supports PDF upload in order to align with the Onfido API.
```
Comment (hjheath) [inline CHANGELOG.md]: Minor: prefer `Live photo upload` or `Selfie upload`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,20 +1,36 @@
...
 
+## [1.0.0]
+
+We are bumping version to 1.0.0 because:
+* SDK has already been implemented in production integrations
+* changes introduced in this version are not backwards-compatible
+* we are following Semantic Versioning for consistency
+
+### Changed
+- Public: Support uploading captures to the Onfido API through use of new SDK tokens (JWT v2)
```
Comment (hjheath) [inline CHANGELOG.md]: Minor: `captures` is internal jargon. Use `documents and live photos`

File: CHANGELOG.md (hunk 3 above)
Comment (dawiddylowicz) [inline CHANGELOG.md]: Ye, I found it like that and felt weird too. I'll update

File: CHANGELOG.md (hunk 2 above)
Comment (dawiddylowicz) [inline CHANGELOG.md]: Good point, updating!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://183-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 6]
```diff
@@ -138,30 +153,25 @@ Congratulations! You have successfully started the flow. Carry on reading the ne
...
 
 - **`onFaceCapture {Function} optional`**
 
-  Callback that fires when the face has been successfully captured and confirmed by the user. It returns an object that contains the face capture.
+  Callback that fires when the face has been successfully captured, confirmed by the user and uploaded to the Onfido API. It returns an object that contains the face capture.
 
 - **`onComplete {Function} optional`**
 
-  Callback that fires when both the document and face have successfully been captured. It returns an object that contains both captures. This event data should sent to your backend where the full [API requests](#uploading-files-to-onfido-api) will be made.
+  Callback that fires when both the document and face have successfully been captured and uploaded. It returns an object that contains all captures. At this point you can trigger your backend to create a check by making a request to the [create check endpoint](https://documentation.onfido.com/#create-check)(https://documentation.onfido.com/#create-check).
```
Comment (stefaniacardenas) [inline README.md]: For some reason, this link is repeated twice `(https://documentation.onfido.com/#create-check)`

Comment (dawiddylowicz) [inline README.md]: Nicely spotted! Correcting

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-183-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://183-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**